worldmap.html

```diff
@@ -211,6 +358,77 @@
             $("#node-input-cluster").spinner({min:0, max:19});
         }
     });
+
+    RED.nodes.registerType('ui_worldmap',{
+        category: 'dashboard',
+        color: 'rgb( 63, 173, 181)',
+        defaults: {
+            group: {type: 'ui_group', required:true},
+            order: {value: 0},
+            width: {
+                value: 0,
+                validate: function(v) {
+                    var valid = true
+                    var width = v||0;
+                    var currentGroup = $('#node-input-group').val()|| this.group;
+                    var groupNode = RED.nodes.node(currentGroup);
+                    valid = !groupNode || +width <= +groupNode.width;
+                    $("#node-input-size").toggleClass("input-error",!valid);
+                    return valid;
+                }},
+            height: {value: 0},
+            name: {value:""},
+            lat: {value:""},
+            lon: {value:""},
+            zoom: {value:""},
+            layer: {value:""},
+            cluster: {value:""},
+            maxage: {value:""},
+            usermenu: {value:"show"},
+            layers: {value:"show"},
+            panit: {value:"false"},
+            panlock: {value:"false"},
+            zoomlock: {value:"false"},
+            hiderightclick: {value:"false"},
+            coords: {value:"false"},
+            showgrid: {value:"false"},
+            path: {value:"/worldmap"}
+        },
+        inputs:1,
+        outputs:0,
+        icon: "white-globe.png",
+        align: "right",
+        label: function() {

```

Comment:
I think we also want a paletteLabel of "worldmap" - It can be called the same as the other one as it is a different colour and in the ui section.

Reply:
changed paletteLabel to `worldmap`.